src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java

```diff
@@ -56,7 +59,7 @@ public static Module getSimpleModule()
                 if ( StreamSupport.stream( beanDescription.getClassInfo().memberMethods().spliterator(), true).
                                 anyMatch( m -> m.hasAnnotation( JsonPostDeserialize.class ) ) )
                 {
-                    logger.debug( "BeanDescription {} ", beanDescription.getClassInfo());
+                    logger.info( "BeanDescription {} ", beanDescription.getClassInfo());
```

Comment:
Same here, debug seems to be fine.

Reply:
Agreed

Reply:
Reverted to debug ; will deal with Jenkins separately.